checks/evaluation/webhooks.go

```diff
@@ -15,47 +15,39 @@
 package evaluation
 
 import (
-	"fmt"
-
 	"github.com/ossf/scorecard/v4/checker"
 	sce "github.com/ossf/scorecard/v4/errors"
 	"github.com/ossf/scorecard/v4/finding"
+	"github.com/ossf/scorecard/v4/probes/webhooksWithoutTokenAuth"
 )
 
 // Webhooks applies the score policy for the Webhooks check.
-func Webhooks(name string, dl checker.DetailLogger,
-	r *checker.WebhooksData,
+func Webhooks(name string,
+	findings []finding.Finding, dl checker.DetailLogger,
 ) checker.CheckResult {
-	if r == nil {
-		e := sce.WithMessage(sce.ErrScorecardInternal, "empty raw data")
-		return checker.CreateRuntimeErrorResult(name, e)
+	expectedProbes := []string{
+		webhooksWithoutTokenAuth.Probe,
 	}
 
-	if len(r.Webhooks) < 1 {
-		return checker.CreateMaxScoreResult(name, "no webhooks defined")
+	if !finding.UniqueProbesEqual(findings, expectedProbes) {
+		e := sce.WithMessage(sce.ErrScorecardInternal, "invalid probe results")
+		return checker.CreateRuntimeErrorResult(name, e)
 	}
 
-	hasNoSecretCount := 0
-	for _, hook := range r.Webhooks {
-		if !hook.UsesAuthSecret {
-			dl.Warn(&checker.LogMessage{
-				Path: hook.Path,
-				Type: finding.FileTypeURL,
-				Text: "Webhook with no secret configured",
-			})
-			hasNoSecretCount++
-		}
-	}
+	totalWebhooks := findings[0].Values["totalWebhooks"]
+	webhooksWithoutSecret := findings[0].Values["webhooksWithoutSecret"]
```

Comment:
will need to adjust if doing multiple findings

Reply:
(and please be mindful of https://github.com/ossf/scorecard/issues/3641 when it comes to these values)

Reply:
`webhooksWithoutSecret` is no longer in the `Values` with https://github.com/ossf/scorecard/pull/3522/commits/cfb0caf8ef483fec819e50dd02eeb0191bd27a1b